scripts/variantstore/beta_docs/gvs-cleanup.md

```diff
@@ -0,0 +1,14 @@
+# Genomic Variant Store Cleanup
+
```

Comment:
Could we include a brief preface here that covers something to the effect of

Running the GVS leaves behind intermediate files, big query tables of variant data, and the final variant outputs. These data accrue approximately $X/genome/month in storage costs. Here, we introduce steps to clean up those outputs to reduce unnecessary storage cost if you no longer need them. 

^?